YelpAPI.podspec

```diff
@@ -27,6 +27,6 @@ Pod::Spec.new do |s|
 
   s.source_files = "Classes/**/*.{h,m}"
 
-  s.public_header_files = 'Classes/**/*.h'
   s.dependency 'TDOAuth', '~> 1.1'
+  s.private_header_files = 'Classes/**/*Private.h'
```

Comment:
Tiny nit: might make sense to move this line right underneath the `source_files` line, then all the lines dealing with our source files are right next to each other. (If one's changed, the other might need to be changed.)